src/Test.UnitTests.Sarif.Multitool/MergeCommandTests.cs

```diff
@@ -24,13 +24,16 @@ namespace Microsoft.CodeAnalysis.Test.UnitTests.Sarif.Multitool
     // requires one or more input files, we'll have to address this. We will probably have to
     // factor the output file writing and comparison infrastructure out of FileDiffingUnitTests.
     // We might be able to push it into a lower base class.
-    public class MergeCommandTests : FileDiffingUnitTests
+    public class MergeCommandTests : FileDiffingUnitTests, IClassFixture<MergeCommandTests.MergeCommandTestsFixture>
```

Comment:
>MergeCommandTests [](start = 17, length = 17)

This change has the effect of removing this class's test output directory before the tests in this class are run, which is useful because otherwise you have files left behind from previous runs. Other Multitool command test classes already had this machinery but I neglected to include it when I created this class yesterday.